### PR TITLE
[Test] Making reset optional

### PR DESCRIPTION
**Pull request by srijan-sivakumar**
Resetting the volume options seems to be a costly transaction. Hence making it optional.

### CODE REVIEW

File: tests/d_parent_test.py [hunk 1]
```diff
@@ -110,7 +111,9 @@ def terminate(self):
             self.redant.logger.error(tb)
 
         try:
-            self.redant.reset_volume_option('all', 'all', self.server_list[0])
+            if self.reset_option:
```
Comment (nik-redhat) [inline tests/d_parent_test.py:114]: Aree....but the TCs need to be updated for this :thinking: , to actually allow cluster reset.

Comment (srijan-sivakumar) [reply tests/d_parent_test.py:114]: Yup..but I see not all cases needs a option reset. And this seems to be the root of failure of the env itself yesterday...

Comment (srijan-sivakumar) [reply tests/d_parent_test.py:114]: We will take this as something to be done by user in a test case. That way..we can prevent a costly operation. With time, we can replace this with a volds based modification.

Comment (nik-redhat) [reply tests/d_parent_test.py:114]: Yeah, that's what I am saying :sweat_smile: , so now if some cases are setting cluster level option it won't be reset, if we do so and not update the TCs in the same patch

Comment (srijan-sivakumar) [reply tests/d_parent_test.py:114]: Will take that as a next patch. I do see 2-3 cases setting cluster level options. But most are setting only volume level options.

**Review by nik-redhat — APPROVED** [review]

**Pull request merged**